src/transformers/__init__.py

```diff
@@ -1090,8 +1090,8 @@
         "is_torch_available",
         "is_torch_neuroncore_available",
         "is_torch_npu_available",
-        "is_torch_tpu_available",
         "is_torchvision_available",
```

Comment:
We need to keep whilst it's still being deprecated

```suggestion
        "is_torch_tpu_available",
        "is_torchvision_available",
```

Reply:
Done